bot.py

```diff
@@ -264,82 +261,93 @@ async def add_summoner(ctx, *, message):
         async with ctx.typing():
             await asyncio.sleep(1)
 
-        # create a directory containing json file to store data for added summoners
-        if not os.path.exists(json_path):
-            os.makedirs(data_folder_path, exist_ok=True)
-            with open(json_path, "w"):
-                pass
-
         # converting the message into list of summoners
-        player_list = message.split(",")
+        # Split by ',' and remove leading/trailling white spaces.
+        user_input_names = [x.strip() for x in message.split(",")]
```

Comment:
@jung1411 Found bug - I'm not sure if it was intentional. 
If user put in; 
`add d4022, leonaldu, orai  ,  deletethisgame`
The code was looking at ` leonaldu` ,` orai  ` (notice the whitespace).

Not sure if this was intentional or not- can you confirm